src/core/dpl-cms/model/dplOpeningHoursGETFormat.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Generated by orval v6.8.1 🍺
```

Comment:
Please consider if this and other files like this are still relevant.

They are marked with the old release of Orval. Consequently it seems as if they have not been created/overridden after upgrading. This makes me wonder if they are still being used.

This goes for a good deal of the new files here.